### PR TITLE
Create VSC 1.24 blog post

### PR DESCRIPTION
**Pull request by iskyOS**

### CODE REVIEW

**Review by jdorfman — APPROVED** [review]
LGTM!

File: content/blogposts/2024/cody-vscode-1-24-0-release.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+We’ve beautified the chat UI with new context chips. The context you @-mention will appear in the chat window with a chip and a symbol indicating the type of context it is (like repo, file, symbol, or webpage context).
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1-24-release/context-chips.png"
+  alt="Context chips in the chat UI"
+/>
+
+## ICYMI: Claude 3.5 Sonnet is here
+
+Last week, we [released Claude 3.5 Sonnet for Cody Pro](https://sourcegraph.com/blog/claude-3.5-sonnet-now-available-in-cody). Cody supports the **best models**, and Anthropic’s latest flagship is already showing [impressive benchmarks](https://www.anthropic.com/news/claude-3-5-sonnet), surpassing Claude 3 Opus in some benchmarks while being far faster. Cody Pro users can now select Claude 3.5 Sonnet for chat and commands (this was shipped in VS Code v1.22.4, but is also available in v1.24.0).
```
Comment (kalanchan) [inline content/blogposts/2024/cody-vscode-1-24-0-release.md:52]: @iskyOS can we add a quick blurb about it coming soon for JetBrains in the following days? I think the version should be `6.0.9`

File: content/blogposts/2024/cody-vscode-1-24-0-release.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+We’ve beautified the chat UI with new context chips. The context you @-mention will appear in the chat window with a chip and a symbol indicating the type of context it is (like repo, file, symbol, or webpage context).
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1-24-release/context-chips.png"
+  alt="Context chips in the chat UI"
+/>
+
+## ICYMI: Claude 3.5 Sonnet is here
+
+Last week, we [released Claude 3.5 Sonnet for Cody Pro](https://sourcegraph.com/blog/claude-3.5-sonnet-now-available-in-cody). Cody supports the **best models**, and Anthropic’s latest flagship is already showing [impressive benchmarks](https://www.anthropic.com/news/claude-3-5-sonnet), surpassing Claude 3 Opus in some benchmarks while being far faster. Cody Pro users can now select Claude 3.5 Sonnet for chat and commands (this was shipped in VS Code v1.22.4, but is also available in v1.24.0). JetBrains users will also get access to Claude 3.5 Sonnet in the coming days
```
Comment (abeatrix) [inline content/blogposts/2024/cody-vscode-1-24-0-release.md:52]: 3.5 Sonnet is also available for Cody Free users

**Review by abeatrix — APPROVED** [review]
Left a comment in-line. I think we should also mention the free models we now offer to Cody Free users: https://github.com/sourcegraph/cody/pull/4665/files

**Pull request merged**